templates/repo/issue/view_content/comments.tmpl

```diff
@@ -697,7 +697,11 @@
 				</span>
 			</div>
 		{{else if and (eq .Type 29) (or (gt .CommitsNum 0) .IsForcePush)}}
-			<div class="timeline-item event" id="{{.HashTag}}">
+			<!-- If PR is closed, the comments whose type is CommentTypePullRequestPush(29) after latestCloseCommentID won't be rendered. //-->
+			{{if and .Issue.IsClosed (gt .ID $.LatestCloseCommentID)}}
+				{{continue}}
+			{{end}}`
+			<div class="timeline-item event" id="{{.HashTag}}">`
```

Comment:
Some stray backticks ended up here, they can be seen in the screenshots in the description.

Reply:
https://github.com/go-gitea/gitea/pull/23282